config/README.md

```diff
@@ -125,7 +125,7 @@ mode = summarize
 ```
 </details>
 
-You could summarize *log_file1* and every log_file, that's found inside *log_directory1* just by:
+You could summarize (or analyze?) *log_file1* and every log_file, that's found inside *log_directory1* just by:
```

Comment:
I have thought about this for a while. I suggest to completely get rid of the --mode flag. Instead it would make sense, to remove the summarize mode ( because it's one special case), which could be implemented in the analysed-summary mode, which also need a new name. 
The possibility of analyzing the files one by one, could then be implemented by a --one-by-one flag (better name suggestions are welcome)

Reply:
This should be assigned in a PR resolving #112